src/hacbs/components/ApplicationDetails/ApplicationQuickstartContent.ts

```diff
@@ -0,0 +1,50 @@
+export const applicationQuickstartContent = {
+  apiVersion: 'console.openshift.io/v1',
+  kind: 'QuickStarts',
+  metadata: {
+    name: 'hacbs-getting-started-app',
+  },
+  spec: {
+    version: 4.7,
+    displayName: `Getting started with an app`,
+    durationMinutes: 60,
+    description: `Getting started with an app.`,
+    introduction: `You created an app and added a component to it. Good job! To get started using your app, follow the steps in this quick start guide.`,
+    tasks: [
+      {
+        title: `Add components and builds`,
+        description: `
+1. To add components and builds, go to the **Actions** dropdown and select "Add Component".
+2. Add the components you want included, then you will receive a PR to add the build pipeline to your code.
+3. View the Pull Requests we have sent you by clicking on the **Merge Pull Requests** button. You will be directed to the Pull Requests list.
+4. Customize this build pipeline according to your development needs.
+5. Merge to the PR to add the build pipeline to your code.`,
+      },
+      {
+        title: `Add integration test`,
+        description: `Integration test is a pipeline, set from GitHub, with the goal to test each component individually and then once more together as one application. Because it is unique per app, this task is on you. But we can help you get there. Check out our guide on how to set it up!  
+            \
+            [Learn about integration tests](http://www.redhat.com)
+1. To add an integration test pipeline, go to the **Action** dropdown of your application and select "Add integration test".
+2. In the URL line, add a URL to your integration test pipeline.
+3. To ensure your app is only released when the test passes, mark the integration test as **Mandatory for Release**.`,
+      },
+      {
+        title: `Manage environments`,
+        description: `Now, we need to deploy your builds somewhere. Add a static environment, an environment available only on this workspace. This will be your development environment.  
+            [Learn more about Static Environments](http://www.redhat.com)
+1. To add a static environment, go to the **Actions** dropdown of your application and select **Add environment**. 
+2. On the next page, select a static environment.
+3. On the Create Static environment page, complete all the mandatory input.
+4. Click **Create**.`,
+      },
+    ],
+    conclusion: `Congratulations! You're all set up.
+        <br><br>
+        Now it's time to:  
+          * Connect to a managed environment to release your app to  
+          * Push some code, monitor its process
+        <br><br>
+        If you get lost, this guide is here to help. You can access it at any time from the **Actions dropdown**.`,
```

Comment:
The list isn't rendered as an HTML list. Also updated the bold `Application Actions` to omit `dropdown`

```suggestion
        Now it's time to:  

* Connect to a managed environment to release your app to  
* Push some code, monitor its process

If you get lost, this guide is here to help. You can access it at any time from the **Application Actions** dropdown.`,
```

Reply:
there is no "Application Actions" dropdown, it's Actions... so i did unbold dropdown but only bolded Actions. it's one of the only overrides i made on the copy, aside from misspellings, although there are plenty of others i ignored.